### ISSUE TITLE
intellisense not working

### ISSUE THREAD

**Issue by irohitb** [microsoft/vscode#90287]
```
Version: 1.42.0
Commit: ae08d5460b5a45169385ff3fd44208f431992451
Date: 2020-02-06T10:51:33.119Z
Electron: 6.1.6
Chrome: 76.0.3809.146
Node.js: 12.4.0
V8: 7.6.303.31-electron.0
OS: Darwin x64 19.2.0
```

### Steps to Reproduce:
So Previously my autocomplete when writing import statements and cmd + click to go to definition used to work properly but after formatting my mac and upgrading to the latest OS, my intellisense have stopped working for any repo (ts and non-ts project) 

![ezgif com-video-to-gif](https://user-images.githubusercontent.com/32276134/74084056-ebcc4400-4a90-11ea-807d-20ae16ebc3b8.gif)

1. In the above gif when I click on the definition/or import path, it won't take me to that file/function (go to definition)
2. When I do `import foo from "{something}`. in the place for something when I try to reference the file it used to give me suggestions based on the folders/files but that doesn't seem to be working as well (auto complete imports)

My settings.json

```
{
    "editor.tabCompletion": "on",
    "emmet.showSuggestionsAsSnippets": true,
    "editor.suggestSelection": "first",
    "editor.multiCursorModifier": "alt",
}
```

### What have I tried. 
1. Disabling all my plugins 
2. javaScript and TypeScript Nightly build extension (just installed it) 
3. I went to version 1.36 in which everything was working, then through mac preference I decided to update the VS-Code to 1.42 and everything was broken 

Does this issue occur when all extensions are disabled?: Yes

**Comment by waelajaber** [comment]
having the exact same issue

**Comment by mjbvz** [comment]
Please provide self-contained repo with a step-by-step explanation of how to reproduce the problem

**Comment by irohitb** [comment]
@mjbvz 
1. it was happening in every repo 
2. There is a GIF in my issue to explain everything (I am not sure about step-by-step explanation to reproduce the problem) 

Anyway, If this helps, I moved back to version 1.37 and started using it. sometime in between VS code would have auto-updated to the latest version. Now I am on version 1.42 and everything is working. 

The trick here for me was to downgrade to 1.37 and work until VS code update by itself. Manually Updating from preference wasn't of any help. 

Matt, I no longer have the problem but not sure if I should close the issue or not.

**Comment by mjbvz** [comment]
Ok, closing this for now since it is no longer happening

Gifs are really hard to follow for issue anything more than the most basic issue reports (and you can't copy text from them!), so in the future please try to use the following structure when reporting issues:

**Steps to reproduce**
1. Starting in state
2. Do X
3. Do Y
4. ...

**Expected**
What should have happened

**Actual**
What actually happened
